### PR TITLE
[Ready(really)]Tileset update

### PR DESCRIPTION
**Pull request by SzQ1**
Updating the UDP with work we have done in https://github.com/Theawesomeboophis/UndeadPeopleTileset not including mods for now except of Aftershock.
Warning, many screenshots below.

Some files have less sprites than previously, I will investigate that. Game looks normal despite of that 
obraz
obraz

Included #1102, #1118, #1655
obraz

Fixes to rotation of map tiles corresponding to #1762  
obraz
obraz

Many new map tiles
obraz

### CODE REVIEW

**Comment by olanti-p** [comment]
A small detail that's somewhat hard to see and very hard to unsee: not all tiles in UDP overmap can rotate cleanly.
Guess that's another reason for explicit rotations.
image

**Comment by SzQ1** [comment]
Good find, will fix.

**Comment by SzQ1** [comment]
`Some files have less sprites than previously, I will investigate that.`
They were moved to different files.
I added snow tiles mod to the repo, had to do it since some time.
Grass issue at Refugee Centre mentioned by Olanti I will fix at other time.

**Comment by SzQ1** [comment]
So i linted the tile_config.json with https://jsonlint.com/ hoping that it will pass the json validation. @Coolthulhu not sure why it failed after that, any suggestion?

**Comment by olanti-p** [comment]
Every online linting tool has its own linting style, different from our own linter.
You need to use either DDA's online linter (should be compatible with BN, [link](https://dev.narc.ro/cataclysm/format.html)) or BN's own `json_formatter.exe` (it's included with every release of the game, just drag your `.json` and drop it onto the exe and its done).

File: data/mods/UDP_BN_FAKE_SNOW/modinfo.json [hunk 1]
```diff
@@ -0,0 +1,11 @@
+[
+  {
+    "type": "MOD_INFO",
+    "ident": "UDP_BN_FAKE_SNOW",
```
Comment (olanti-p) [inline data/mods/UDP_BN_FAKE_SNOW/modinfo.json:4]:
```suggestion
    "id": "UDP_BN_FAKE_SNOW",
```

**Pull request merged**